TACC/Core-Portal
# Bug/FP-1080: Fix search loading

## Overview: ##
Hides previous search results on loading next result.

## Related Jira tickets: ##

* [FP-1080](https://jira.tacc.utexas.edu/browse/FP-1080)

## Summary of Changes: ##
Fixes conditionals in the `SiteSearchListing` component so that we don't show previous search results when
search is in progress for next results.

## Testing Steps: ##
1. Search for something.
2. Search for something else.
3. Ensure that the result of the previous search is not present with the loading spinner.

## UI Photos:
N/A

## Notes: ##
